### ISSUE TITLE
Timer doesn't run when page is inactive

### ISSUE THREAD

**Issue by tslater2006** [chrisxclash/perk-watch#1]
I would like to be able to keep perkpoints open during the day while I work, this usually involves multiple tabs being open. It seems once perkpoints.xyz tab loses focus the timer pauses. Not sure if something can be done about this. If not just close this I guess :)

Thanks for the wonderful tool regardless.

**Comment by chrisxclash** [comment]
Yeah, I noticed this a while ago. It's due to how browsers don't run intervals/timers when the tab isn't in focus. The best temporary solution you can do is just open the site in it's own window in your browser (ctrl +n) and that way even if it's not the active window since it's the active tab it should keep going.

I might look into doing web workers at some point to have it always work.

And no worries, glad you like it!
